bayesian/utils.py

```diff
@@ -1052,11 +1052,19 @@ def get_recommendation_feedback_by_ecosystem(ecosystem):
             filter(RecommendationFeedback.stack_id == StackAnalysisRequest.id).all()
         result = []
         for feedback in feedback_list:
+            if feedback.stack_request.dep_snapshot.result is None:
```

Comment:
I think this should be `if feedback.stack_request.dep_snapshot is None`. I.e. without `.result` part.

Reply:
Or, even better, `if not feedback.stack_request.dep_snapshot:`